identity-issuance/src/main/java/com/android/identity/issuance/hardcoded/IssuingAuthorityState.kt

```diff
@@ -387,6 +396,23 @@ class IssuingAuthorityState(
     }
 
     private fun createPresentationData(
+        env: FlowEnvironment,
+        format: CredentialFormat,
+        documentConfiguration: DocumentConfiguration,
+        authenticationKey: EcPublicKey
+    ): ByteArray {
+        when (format) {
+            CredentialFormat.MDOC_MSO -> {
+                return createPresentationDataMdoc(env, documentConfiguration, authenticationKey)
+            }
+            CredentialFormat.SD_JWT_VC -> {
+                return createPresentationDataSdJwt(env, documentConfiguration, authenticationKey)
+            }
+            else -> throw IllegalArgumentException("Unsupported credential format $format")
+        }
+    }
```

Comment:
Can be simplified to 

```
private fun createPresentationData(
    env: FlowEnvironment,
    format: CredentialFormat,
    documentConfiguration: DocumentConfiguration,
    authenticationKey: EcPublicKey
): ByteArray = 
    when (format) {
        CredentialFormat.MDOC_MSO -> createPresentationDataMdoc(env, documentConfiguration, authenticationKey)
        CredentialFormat.SD_JWT_VC -> createPresentationDataSdJwt(env, documentConfiguration, authenticationKey)
        else -> throw IllegalArgumentException("Unsupported credential format $format")
    }
```

Reply:
Fixed, also dropped the `else` branch which is redundant.